gui/default/index.html

```diff
@@ -126,7 +126,7 @@ <h3 class="panel-title">
               <li ng-if="authenticated" class="divider" aria-hidden="true"></li>
               <li ng-if="authenticated"><a href="" ng-click="advanced()"><span class="fa fa-fw fa-cogs"></span>&nbsp;<span translate>Advanced</span></a></li>
               <li ng-if="authenticated"><a href="" ng-click="logging.show()"><span class="fa fa-fw fa-wrench"></span>&nbsp;<span translate>Logs</span></a></li>
-              <li ng-if="authenticated"><a href="" ng-click="logout()"><span class="far fa-fw fa-ban"></span>&nbsp;<span translate>Log Out</span></a></li>
+              <li ng-if="authenticated && isAuthEnabled(config)"><a href="" ng-click="logout()"><span class="far fa-fw fa-ban"></span>&nbsp;<span translate>Log Out</span></a></li>
```

Comment:
The config we're passing is `$scope.config`, right, so we might as well not pass it as a parameter and have isAuthEnabled do the needful directly?

Reply:
Yeah, fair enough. The function is also called in `refreshNoAuthWarning()`, but that also just uses `$scope.config` too.